pypeit/scripts/show_1dspec.py

```diff
@@ -35,34 +35,47 @@ def main(args, unit_test=False):
     from pypeit import specobjs
     from pypeit import msgs
 
-    from IPython import embed
-
-    sobjs = specobjs.SpecObjs.from_fitsfile(args.file)
-
-    # List only?
-    if args.list:
-        print("Showing object names for input file...")
-        for ii in range(len(sobjs)):
-            name = sobjs[ii].name
-            print("EXT{:07d} = {}".format(ii+1, name))
-        return
-
-    # Load spectrum
-    if args.obj is not None:
-        exten = sobjs.name.index(args.obj)
-        if exten < 0:
-            msgs.error("Bad input object name: {:s}".format(args.obj))
-    else:
-        exten = args.exten-1 # 1-index in FITS file
-
-    # Check Extraction
-    if args.extract == 'OPT':
-        if 'OPT_WAVE' not in sobjs[exten]._data.keys():
-            msgs.error("Spectrum not extracted with OPT.  Try --extract=BOX")
+    try:
```

Comment:
We need a command-line argument, not this try except block.

Reply:
See my comment above in response's to @profxj point about the same try/except.